File: javaScript/Gate Two/TaskTen.js
const input = require("prompt-sync")(); 

let userNumber = input("Enter a number: ");
let number = userNumber;
let count = 0;

while(userNumber >= 1){
	count += 1;
	userNumber /= 10;
	}
console.log(`There are ${count} digits in the number ${number}`);
	

File: java/Gate Two/TaskTen.java
import java.util.Scanner;
 
public class TaskTen{
    public static void main(String[] args){
	Scanner userInput = new Scanner(System.in);
	System.out.print("Enter a number: ");
	int userNumber = userInput.nextInt();
	int number = userNumber;
	int count = 0;

	while(userNumber != 0){
		count += 1;
		userNumber  = userNumber / 10;
		}
     	System.out.printf("There are %d digits in the number %d \n", count, number)
;
	}
}
